### DISCUSSION TITLE
Get logs in executed migration ?

### DISCUSSION THREAD

**Discussion by MrVhek** [prisma/prisma#22171] [Q&A] [answered]
### Question

I am currently making a custom prisma migration for migration needs. In this SQL migration, I do put some RAISE NOTICE to have information on what's going on and ensure it has all been done correctly. But from prisma migrate dev side, I don't have any of these logs. Is there any way to activate it ?

Thanks for your help !

### How to reproduce (optional)

Create a new SQL migration (postgres), do a RAISE NOTICE 'test', nothing will happen when running npx prisma migrate dev

### Expected behavior (optional)

Have SQL RAISE NOTICE logs on terminal

### Information about Prisma Schema, Client Queries and Environment (optional)

No schema involved

- OS: macOS
- Database: PostgreSQL
- Node.js version: 18.18.2
- Prisma version : 5.5.2

**Comment by ludralph** [comment, marked as answer]
Hi @MrVhek 👋 

Thank you for raising this question.

There is currently no direct support for capturing `RAISE NOTICE` messages from PostgreSQL functions in Prisma. This has been previously requested in this open issue below although within the context of a Prisma TransactionClient.

- https://github.com/prisma/prisma/issues/16324

Prisma does not provide a built-in way to listen to PostgreSQL notices or warnings during migrations. The standard logging functionality of Prisma, which can be enabled by setting the log option in the PrismaClient constructor, does not include capturing `RAISE NOTICE` messages from PostgreSQL. As of now, Prisma does not provide a built-in way to listen to PostgreSQL notices or warnings during migrations. 

You can document your use case in the open issue and add a 👍  to help our engineers prioritize it.

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏
